CMakeLists.txt

```diff
@@ -1,36 +1,36 @@
-# Copyright (c) BBP/EPFL 2011-2015 Stefan.Eilemann@epfl.ch
+# Copyright (c) BBP/EPFL 2016 Stefan.Eilemann@epfl.ch
 
-# Never mind the [] sections. They are only here for doxygen.
-# Remove them in your project.
-
-# //! [Setup]
 cmake_minimum_required(VERSION 3.1 FATAL_ERROR)
 project(Keyv VERSION 1.0.0)
 set(Hello_VERSION_ABI 1)
 
 list(APPEND CMAKE_MODULE_PATH ${PROJECT_SOURCE_DIR}/CMake
                               ${CMAKE_SOURCE_DIR}/CMake/common)
 include(GitExternal)
-# //! [Setup]
 
-# //! [Scripts]
-set(HELLO_DESCRIPTION "Example project for the Blue Brain Project")
-set(HELLO_MAINTAINER "Blue Brain Project <bbp-open-source@googlegroups.com>")
-set(HELLO_LICENSE LGPL BSD)
+set(KEYV_DESCRIPTION
+  "Key-Value based storage and caching using a variety of backends")
+set(KEYV_MAINTAINER "Blue Brain Project <bbp-open-source@googlegroups.com>")
+set(KEYV_LICENSE LGPL BSD)
```

Comment:
Yes, but IMHO it would be clearer to indicate only the license for the main library. The fact that some extra files have a separate license is not so important at this point for prospective users, just a bit confusing. People kind of expect to be free to reuse the code examples. There could also be icons under CreativeCommons, config files without license, etc... we don't want to list everything. What do you think?